pkg/controller/controller.go

```diff
@@ -419,9 +422,20 @@ func New(ctx context.Context, cfg *config.Config, kubeclientset kubernetes.Inter
 
 	controller.levelMonitor = NewResourceLevelMonitor(scope.NewSubScope("collector"), flyteworkflowInformer.Lister())
 
+	// The admin client might not be initialized if EnableAdminLauncher is set to False.
+	if adminClient == nil {
+		var err error
+		adminClient, err = getAdminClient(ctx)
```

Comment:
If we now always want an admin client, let's remove the condition and just always initialize it and use it up there if needed and down here all the time...

Reply:
done